### PR TITLE
Add game type and env entry point to agent's obs

### PR DESCRIPTION
**Pull request by weseldridge**

### CODE REVIEW

File: pommerman/forward_model.py [hunk 1]
```diff
@@ -532,12 +533,13 @@ def in_view_range(position, v_row, v_col):
             bomb_blast_strengths, bomb_life = make_bomb_maps(agent.position)
             agent_obs['bomb_blast_strength'] = bomb_blast_strengths
             agent_obs['bomb_life'] = bomb_life
+            agent_obs['game_type'] = game_type.name
```
Comment (cinjon) [inline pommerman/forward_model.py]: 'name' or 'value'? the latter will be an int and probably easier to use given it's an enum.

**Comment by cinjon** [comment]
thanks for doing this Wes.

File: pommerman/forward_model.py (hunk 1 above)
Comment (weseldridge) [reply pommerman/forward_model.py]: yeah. that makes sense.

File: pommerman/cli/run_battle.py [hunk 2]
```diff
@@ -65,6 +65,9 @@ def _run(record_pngs_dir=None, record_json_dir=None):
             actions = env.act(obs)
             obs, reward, done, info = env.step(actions)
 
+            if step is 1:
```
Comment (cinjon) [inline pommerman/cli/run_battle.py]: pls delete.

File: pommerman/configs.py [hunk 3]
```diff
@@ -31,6 +31,7 @@ def ffa_competition_env():
         'num_items': constants.NUM_ITEMS,
         'max_steps': constants.MAX_STEPS,
         'render_fps': constants.RENDER_FPS,
+        'env': 0,
```
Comment (cinjon) [inline pommerman/configs.py]: i think we should be deliberate and say the full entry point, e.g. `pommerman.envs.v0:Pomme`. That will be more configurable as well as clearer.

File: pommerman/cli/run_battle.py (hunk 2 above)
Comment (weseldridge) [reply pommerman/cli/run_battle.py]: Yeah. It will not be there when it comes out of WIP. :D

**Comment by cinjon** [comment]
left two comments, otherwise looks g2g.
(fix those, then squash these together with `git rebase -i`, then let's get this on the road)

**Comment by weseldridge** [comment]
Ok. This should be good to go now. 👌

**Comment by cinjon** [comment]
lgtm.

**Pull request merged**